src/main/java/org/broadinstitute/hellbender/tools/spark/sv/RunSGAViaProcessBuilderOnSpark.java

```diff
@@ -524,6 +540,15 @@ public String toString(){
             return StringUtils.join(toListOfStrings(),"\n");
         }
 
+        /**
+         * Pack the entire fasta record on one line so that it's easy for downstream
+         * Spark tools to parse without worrying about partitioning.
+         * @return
```

Comment:
return is undocumented.  either add a description for the tag, or remove the `@return`.